x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.ts

```diff
@@ -241,7 +241,7 @@ export const fetchRulesSnoozeSettings = async ({
   const response = await KibanaServices.get().http.fetch<RulesSnoozeSettingsBatchResponse>(
     INTERNAL_ALERTING_API_FIND_RULES_PATH,
     {
-      method: 'GET',
+      method: 'POST',
```

Comment:
Is that correct? HTTP GET API endpoint is public. Shouldn't we use a public API endpoint instead?

Reply:
I think the issue is that we need to change the code to use `body` instead of `query`. The internal API is similar to the public one plus with more capabilities.